problems/strings.js

```diff
@@ -87,14 +118,33 @@ function removeLastWord() {}
  * @returns {string}
  */
 
-function everyOtherLetter() {}
+function everyOtherLetter(string) {
+  let newstr = ""
+  for(let i=0; i < string.length; i += 2) {
+    newstr += string[i]
+    
+  }
+  return newstr;
+}
 
 /**
  * Takes in a string and returns a new string that has all the vowels removed.
  * @param {string}
  * @returns {string}
  */
-function disemvowel() {}
+function disemvowel(string) {
+  
+  let newstr = "";
+
+  for(let i=0; i < string.length; i++) {
+    // let vowels = string.includes("a", "e", "i", "o", "u", "A", "E", "I", "O", "U")
```

Comment:
Includes would work better. 
let vowels = "aeiou" // outside of for loop

vowels.include(str[i])  // inside of loop